.changeset/nine-fireants-protect.md

```diff
@@ -0,0 +1,5 @@
+---
+"@opennextjs/cloudflare": patch
+---
+
+fix: CustomRequest instantiation
```

Comment:
Since this text is used to create the changelog, I prefer to add some more context to these files.